charts/oidc-proxy/templates/configmap.yaml

```diff
@@ -51,6 +51,9 @@ data:
     # 2. it causes intermittent duplicate requests between the proxy and the underlying app
     # the latter side effect is too dangerous, so cache encryption is off
     OIDCCacheEncrypt Off
+    # Use GET for the introspection endpoint to avoid a performance issue with POST requests.
+    # See https://broadworkbench.atlassian.net/browse/PROD-634
+    OIDCOAuthIntrospectionEndpointMethod GET
```

Comment:
Good to know that with the new mod_oauth we were already using the GET endpoint.